//-----------------------------------------------------------------------------
// {C} Copyright 2017 Pensando Systems Inc. All rights reserved
//-----------------------------------------------------------------------------

#ifndef __DESCRIPTOR_AOL_HPP__
#define __DESCRIPTOR_AOL_HPP__


#include "nic/include/base.hpp"
#include "nic/include/pd.hpp"
#include "gen/proto/internal.pb.h"

using internal::DescrAolSpec;
using internal::DescrAolRequest;
using internal::DescrAolRequestMsg;
using internal::DescrAolResponseMsg;

namespace hal {
}
#endif /* __DESCRIPTOR_AOL_HPP__ */
